# Problem 2: RNA-seq analysis ğŸ§¬
For this problem, we had several data files, such as BAM filesğŸ“„, txt files with read counts and FPKM correctionğŸ“„ and a Differential Expression analysis fileğŸ“„. Some of the steps needed to solve the diverse questions and objectives we were asked to adress were solved with the following:

## ğŸ“ FPKMS_plotting
- ğŸ“„ `Mutant_vs_WildType_results.txt`: file with normalized FPKMs for each replica, with calculated fold change (FC) and p-value.
- ğŸ’» `Box_plot.R`: script that plots the box plotğŸ“ˆ of FPKMs in all the replicas and the average.
- ğŸ’» `Dispersion_diagram.R`: script that plots the dispersion diagramğŸ“ˆ of FPKMs in all the replicas.
- ğŸ¨ `Boxplot.png` and `Dispersion_diagram.png`: images of the Box plot and the Dispersions Diagram. 


## ğŸ“ Volcano_plot
- ğŸ“„ `Mutant_vs_WildType_results.txt`: file with normalized FPKMs for each replica, with calculated fold change (FC) and p-value.
- ğŸ’» `Volcano_plot.R`: script that creates the volcano plotğŸ“ˆ of the fold changes and the corresponding p-values.
- ğŸ¨ `volcano_plot.png`: volcano plot for the Fold Change and p-adj values.  


## ğŸ“ COG_terms_plotting
- ğŸ“„ `DEGs_woth_COGs.csv`: COG term for every differentially expressed gene associated to a locus_tag. 
- ğŸ’» `COG_terms_plots.R`: script that mathches the COG terms to our DEGs, creates a barplot and computes Fisher's exact test on data. 
- ğŸ¨ `Barplot_comparison_significance.png`: barplot for the different COG terms distinguished by DEGs and original count. 
